lib/maths/CBoostedTreeImpl.cc

```diff
@@ -1381,7 +1381,9 @@ void CBoostedTreeImpl::restoreBestHyperparameters() {
 std::size_t CBoostedTreeImpl::numberHyperparametersToTune() const {
     return m_RegularizationOverride.countNotSet() +
            (m_DownsampleFactorOverride != boost::none ? 0 : 1) +
-           (m_EtaOverride != boost::none ? 0 : 2) +
+           (m_EtaOverride != boost::none
```

Comment:
it seems like here in the scenario where eta is overridden but eta_growth is not, we will calculate zero params to tune. Is that right? I would expect we'd count 1 param to tune (eta_growth).